scripts/core/kernel.yml

```
@@ -372,6 +382,9 @@ params:
     - type: "uint32_t"
       name: argIndex
       desc: "[in] argument index in range [0, num args - 1]"
+    - type: "const $x_mem_obj_properties_t*"
```

Mirroring my other suggestion:

```suggestion
    - type: "const $x_kernel_arg_mem_obj_properties_t*"
```

---

fixed here https://github.com/oneapi-src/unified-runtime/pull/571/commits/85393506eda9c7c77fb6c3ca2d36f46c24c5329b